checkov/common/output/sarif.py

```diff
@@ -127,6 +141,12 @@ def _create_cve_rule(self, check_type: str, record: Record) -> dict[str, Any] |
             "defaultConfiguration": {"level": "error"},
         }
 
+        # Adding 'properties' dictionary only if 'record.severity' exists
+        if record.severity:
+            rule["properties"] = {
+                "security-severity": SEVERITY_TO_SCORE.get(record.severity.name.lower(), "0.0"),
+            }
```

Comment:
this is only relevant for CVEs, because IaC and licenses don't have this information, but you should first check the existence of CVSS score under `record.vulnerability_details` and use it instead of artificially mapping it.

```suggestion
        if record.severity:
            cvss = record.vulnerability_details.get("cvss")
            if not cvss:
                cvss = SEVERITY_TO_SCORE.get(record.severity.name.lower(), "0.0")
            rule["properties"] = {
                "security-severity": cvss,
            }
```

Reply:
Text has been updated and added check to get the CVSS score if it exists.